less/forms.less

```diff
@@ -0,0 +1,37 @@
+.contact {
+  width: 100%;
+}
+
+.contact-form {
+.flexy(flex, column, null, center, left);
+  width: 900px;
+  max-width: 100%;
+  padding: 2em 1em 0px 1em;
+  & input,
+  & select,
+  & textarea {
+    margin: 5px 0px;
+  }
+  input {
+    height: 5rem;
+    font-size: 1.6rem;
+  }
+  label {
+    font-size: 1.6rem;
+  }
+  input[type='submit'] {
+    height: 4rem;
+    color: #fff;
+    background-color: #69D36E;
+    border: none;
+    border-radius: 10px;
+    cursor: pointer;
+    &:hover {
+      .pulse();
+    }
+  }
+}
```

Comment:
More of a cosmetic observation, but would you consider applying the pulsing animation to the `:active` pseudoclass instead (it might confuse people into thinking that they clicked on the button by accident)? Perhaps a non-pulsing scale-up animation would be more appropriate for the `:hover` pseudoclass.